apps/theming-designer/src/components/SemanticSlots.tsx

```diff
@@ -432,20 +44,102 @@ export const SemanticSlots: React.StatelessComponent<ISemanticSlotsProps> = (pro
     );
   };
 
-  const fillVariantSlotsList = (variantType: VariantThemeType, mappingType: {}): JSX.Element[] => {
+  const trimPaletteColors = (paletteColors: {}): any => {
+    let trimmedPaletteColors = {};
+    for (let palette in paletteColors) {
+      if (
+        palette.includes('theme') ||
+        palette.includes('neutral') ||
+        palette.includes('black') ||
+        palette.includes('white') ||
+        palette.includes('redDark') ||
+        palette.includes('accent')
+      ) {
+        (trimmedPaletteColors as any)[palette] = (paletteColors as any)[palette];
+      }
+    }
+    return trimmedPaletteColors;
+  };
+
+  const trimSemanticColors = (semanticColors: {}): any => {
```

Comment:
what's the difference between a semanticColor and a semanticSlot? theyre the same thing...